cogs/eventlogger.py

```diff
@@ -0,0 +1,69 @@
+# Std Lib Imports
```

Comment:
prob not have this in this PR until its a more completed feature

Reply:
can do